packages/hardhat-core/src/internal/util/bigint.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  BigIntLike,
+  bufferToBigInt,
+  bufferToHex,
+  toBuffer,
+} from "@ethereumjs/util";
```

Comment:
I'm pretty sure that having this import at the top-level means that we are gonna end up loading ethereumjs/util while booting hardhat, which is slow.

Also, I think we should avoid using ethereumjs here.

Reply:
Fixed in https://github.com/NomicFoundation/hardhat/pull/3082/commits/767ca3f6f7433e595599933ae30963908bbbf992